modules/exploits/unix/webapp/caidao_php_backdoor_exec.rb

```diff
@@ -0,0 +1,95 @@
+
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'              => 'Chinese Caidao PHP Backdoor Command Execution',
+      'Description'       => %q{
+        This module exploits chinese caidao php backdoor which allows
+        os command execution.
+      },
+      'License'           => MSF_LICENSE,
+      'Author'            => ['Nixawk'],
+      'References'        => [['URL', 'http://www.maicaidao.com']],
```

Comment:
What's this website? Chrome is treating this as malicious. If this malicious it's probably not a good idea to put it here.